codec/encoder/core/src/encoder_ext.cpp

```diff
@@ -3907,6 +3907,10 @@ int32_t WelsEncoderEncodeExt (sWelsEncCtx* pCtx, SFrameBSInfo* pFbi, const SSour
         WelsLog (pLogCtx, WELS_LOG_WARNING,
                  "WelsEncoderEncodeExt()MinCr Checking,codec bitstream size is larger than Level limitation");
     }
+
+    if (iSpatialNum == 1 && NULL != pFbi->pReconPic)
```

Comment:
Can we keep these checks inside the `CopyReconPicToBSInfo` functions? We need to ensure that `bHaveRecon` is to `false` since we don't initialize this structure.